### ISSUE TITLE
Opiate Withdrawal has no effects below 20%, and I mean NO effects.

### ISSUE THREAD

**Issue by GlitchAnims** [Regalis11/Barotrauma#10589]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

![imagen](https://user-images.githubusercontent.com/38194089/204115827-e585bb86-c2c6-41b3-bc13-035ab0b2e308.png)

By no effects I mean **nothing**. There is no Effect component for the values between 0 and 20, which means the affliction will never change unless affected by additional chemicals.

What usually occurs in-game, is that you get injected by Morphine, build up a little bit of Withdrawal, and then at some point you are injected by Morphine for a second time. What happens now is that your Withdrawal goes straight to 0 due to the StatusEffect component on the Morphine item. Opiate Addiction then builds up _just above_ the threshold to start inflicting Opiate Withdrawal, but the Addiction very quickly falls below that threshold, and Withdrawal gets stuck somewhere between 0% and 5%, which is below the 'showinhealthscannerthreshold' check. This is why it is never noticed, it's invisible most of the time. If it does happen to go above this, due to repeated use of opiate items, it usually goes high enough that a Medic will _notice_ it's gone high, so they get injected with Stabilozine, therefore removing all of it, including the Addiction. Either that, or they just get injected with morphine again at a later moment, when the Opiate Addiction has gone low enough to reenact the first example I gave. Hell, most of the time it straight up gets removed by an Outpost Clinic.

TL;DR, Opiate Withdrawal has no changes when between 0% and 20% strength, and it's invisible below 5%, which I believe is the main reason this has not been noticed before.

### Reproduction steps

Start a game, inflict opiate withdrawal onto a person through the console for easy testing. As longs as it's below 20%, it will never change.

### Bug prevalence

Happens every time I play

### Version

0.20.8.0 (Unstable)

### Which operating system did you encounter this bug on?

Windows

### Relevant error messages and crash reports

```shell
No crash, no error.
```

**Comment by Regalis11** [comment]
Fixed in https://github.com/Regalis11/Barotrauma-development/commit/1ab021561d56c862364feba2e1012ff7d84376cd

**Comment by Rokvach** [comment]
Tested, no issues. Closing.
